src/Components/Color/ContrastChecker.jsx

```diff
@@ -0,0 +1,23 @@
+async function checkContrast(hex, contrastText) {
+  const response = await fetch(
+    "https://www.aremycolorsaccessible.com/api/are-they",
+    {
+      method: "POST",
+      body: JSON.stringify({
+        foregroundColor: contrastText,
+        backgroundColor: hex,
+      }),
+      headers: {
+        "Content-Type": "application/json",
+      },
+    }
+  );
+
+  if (response.ok) {
```

Comment:
ahh sorry, here you have you error validation! perfect

Reply:
no it is outdated, but i know what you mean. I can include try and catch. 
thanks for the hint.